src/Nethermind/Nethermind.Synchronization.Test/BlockDownloaderTests.Merge.cs

```diff
@@ -470,6 +470,7 @@ public override BlockDownloader BlockDownloader
             get
             {
                 return _mergeBlockDownloader ??= new(
+                    0,
```

Comment:
I don agree with @LukaszRozmej on moving to end and defaulting to 0; otherwise should name the parameter when passing in so you know what `0` means without having to look up the constructor
```suggestion
                    maxNumberOfProcessingThread: 0,
```